setup.py

```diff
@@ -11,7 +11,7 @@
     py_modules=["tap_stripe"],
     install_requires=[
         "singer-python==5.5.1",
-        "stripe==2.10.1",
+        "stripe==2.42.0",
```

Comment:
We should think a bit more about bumping this SDK version for the tap, either putting it in extras_require to make just the tests use this version, or making the tests run on version `2.10.1` to keep them consistent.